### DISCUSSION TITLE
feature request: Folders and Arrange by

### DISCUSSION THREAD

**Discussion by 1bulf** [mshibanami/redirect-web#58] [Ideas]
Hello,

I would like to suggest a feature to organize extensions into folders and to add features to arrange order of displayed items.

When choosing the location for the folders, I think it would be good keep in mind to keep the UI continuity between the apps for iOS and for wide screen.

In the attached picture I show the two options of where to put the folders. In case of the 1st option, the UI should be done on iOS the same way as it is in the Mail App - one screen for folders and one screen for individual items with moving from one to another being done with a swipe. In case of the 2nd option, I think the folders should be displayed on the top, below them a spacer and below them individual redirect rules that are not in a folder.

On the top of the window / screen on iOS, an ’’arrange by’’ button would be great. It would arrange both redirect rules and folders by either name, date created or date changed. There might be an option to arrange folders and redirect rules by different pattern or even to allow for different arrangement pattern within each folder. But I think that would be overly complicated in this case.

Best regards,

Bulf

![91CAB83E-D45C-4C08-BED3-DA4EF3117027](https://github.com/user-attachments/assets/75e0ea7e-7b71-407a-9eaf-f4af0cac832e)

**Comment by mshibanami** [comment]
Thank you for your feedback.

Yes, other users also requested me it through email and it's on my TODO list.
But it will take time to implement it since I didn't plan it and I will need to make breaking changes. I hope you will understand. 🙂

**Comment by jsnow149** [comment]
Would tags be easier to implement? It would be so helpful to organize redirects using folders, tags, etc.

**Reply by MrDunkel089** [reply]
Tags with some kind of folder structure would be awsome!
